build.py

```diff
@@ -64,14 +64,11 @@ def build():
     # native module and discard the rest !
 
     maturin_build_profile = "--profile=" + os.environ.get("CARGO_PROFILE", DEFAULT_CARGO_PROFILE)
-    features = (
-        "--no-default-features --features use-sodiumoxide"
-        if maturin_build_profile == "--profile=release"
-        else ""
-    )
+    maturin_build_features = os.environ.get("CARGO_FEATURES", "")
```

Comment:
This environ variable can be use to inject any parameter, hence we should rename it as `CARGO_EXTRA_ARGS`

Reply:
Done